### PR TITLE
Enhancement/permissions engine

### PR DESCRIPTION
**Pull request by Wardormeur**

### CODE REVIEW

File: config/perm/events.js [hunk 1]
```diff
@@ -9,6 +9,7 @@ module.exports = function () {
       	cmd: 'is_ticketing_admin'
       }]
     }],
+    //  TODO: verify this one, seems meh to me
```
Comment (DanielBrierton) [inline config/perm/events.js]: It's only used for filling out the form when editing an event. That's probably no reason to lock it down to basic-user though...

File: config/perm/events.js [hunk 2]
```diff
@@ -46,8 +47,13 @@ module.exports = function () {
     'saveApplication': [{
       role: 'basic-user'
     }],
+    //  TODO: straighten?
```
Comment (DanielBrierton) [inline config/perm/events.js]: What do you mean by straighten?

Comment (Wardormeur) [inline config/perm/events.js]: Looks like i did it (make it more restricted), outdated comm

File: config/perm/events.js (hunk 1 above)
Comment (Wardormeur) [inline config/perm/events.js]: If events doesn't returns the invited people by default, i'd say yes, didn't fully verified returned value @butlerx  ?

**Pull request merged**